salesforce_bulk/salesforce_bulk.py

```diff
@@ -309,7 +309,7 @@ def query(self, job_id, soql, contentType='CSV'):
                 re.search(re.compile(r"from (\w+)", re.I), soql).group(1),
                 "query", contentType=contentType)
 
-        job_content_type = self.job_content_types[job_id]
+        job_content_type = self.job_content_types[job_id] or contentType
```

Comment:
This will raise an error if it can't find `job_id`. There is a method to get with a default:
```
job_content_type = self.job_content_types.get(job_id, contentType)